os/mm/mm_heap/mm_heapinfo.c

```diff
@@ -55,27 +55,15 @@
 #include <tinyara/config.h>
 #include <tinyara/sched.h>
 #include <tinyara/mm/mm.h>
-#include <tinyara/arch.h>
-#include <unistd.h>
 #include <stdint.h>
 #include <stdio.h>
-#ifdef CONFIG_HEAPINFO_USER_GROUP
-#include <string.h>
-#include <tinyara/mm/heapinfo_internal.h>
-#endif
 
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
 #define HEAPINFO_INT INT16_MAX
 #define HEAPINFO_NONSCHED (INT16_MAX - 1)
 
-#ifdef CONFIG_HEAPINFO_USER_GROUP
-int max_group;
-struct heapinfo_group_s heapinfo_group[HEAPINFO_USER_GROUP_NUM];
-struct heapinfo_group_info_s group_info[HEAPINFO_THREAD_NUM];
-#endif
-
 #if CONFIG_KMM_NHEAPS > 1
```

Comment:
1. This file only has the function, heapinfo_parse. How about using it as a file name?
2. parse what?

Reply:
I changed. Thanks.